python/paddle/v2/framework/tests/op_test.py

```diff
@@ -185,10 +185,9 @@ def check_output_with_place(self, place):
         for out_name, out_dup in Operator.get_op_outputs(self.op.type()):
             if out_dup:
                 sub_out = self.outputs[out_name]
-                for sub_out_name in sub_out:
+                for sub_out_name, expect in sub_out:
```

Comment:
```
for sub_out_name, expect in sub_out.iteritems():
```

Reply:
Done.

Reply:
You can add some type checking code, so that if something went wrong, the python traceback message will be more readable

Reply:
Done.